src/main/java/net/starlark/java/cmd/Main.java

```diff
@@ -86,11 +95,6 @@ private static void readEvalPrintLoop() {
     System.err.println("Welcome to Starlark (java.starlark.net)");
     String line;
 
-    // TODO(adonovan): parse a compound statement, like the Python and
```

Comment:
Please keep this TODO: only the parser is able to know accurately when to stop reading.
(See the history of the go.starlark.net REPL, which is very similar. There were some strange failure modes of the approach implemented in this PR, even though it is strictly an improvement.)